services/postgresql.rst

```diff
@@ -13,9 +13,9 @@ To use it in your application, add it to ``.symfony/services.yaml``:
         type: postgresql:12
         disk: 1024
 
-.. note::
+.. caution::
```

Comment:
I thought about it and forget it. thank you for fixing my mistake :)

Reply:
No problem 🙂👍🏻